mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -0,0 +1,101 @@
+CLASS net/minecraft/unmapped/C_vhpmxtfv net/minecraft/block/MapColor
+	COMMENT Represents the surface color of a block when rendered from the {@link net.minecraft.client.render.MapRenderer}.
```

Comment:
i'd link to the instance in `MinecraftClient` -- the usage of `the` makes it seem like that's the intent